app/src/main/cpp/src/appMain/sample_policy_manager.py

```diff
@@ -50,8 +52,12 @@ def decrypt(data):
     return data
 
 
-def pack(data, encryption, add_http_header):
+def pack(data, encryption, add_http_header, android):
     file_path = data['fileName']
+
+    if android:
+        path = "/tmp/sdl_snapshot.json"
```

Comment:
@VadymLuchko probably I am missing something, but variable `path` is unused in this function. You are using `file_path` here.
Also, maybe it's better to calculate a correct path for all platforms outside the function and replace it in a `data` for android?

Reply:
@AKalinich-Luxoft thanks about "path" I have noticed this typo after push and fixed in next commit
e0fd5100b84e13d8c930043276b9b3f9d0ee5c3f